src/main/java/Domain/UserController.java

```diff
@@ -1,4 +1,5 @@
 package Domain;
+
```

Comment:
_users
_passwordEncoder

Reply:
I didn't change this file, just added the package.